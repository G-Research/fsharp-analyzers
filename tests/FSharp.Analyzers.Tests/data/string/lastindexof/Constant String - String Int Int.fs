module M

"foo".LastIndexOf("p", 0 ,1)
