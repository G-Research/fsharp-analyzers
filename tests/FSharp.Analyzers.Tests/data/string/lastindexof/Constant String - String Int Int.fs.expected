GRA-STRING-004 | Warning | (3,0 - 3,28) | The usage of String.LastIndexOf with a single string argument is discouraged. Signal your intention explicitly by calling an overload. | []
